app/views/tasklist/new.erb

```diff
@@ -0,0 +1,15 @@
+
+<section class= "form">
+  <span><%=form_for @tasklist do |f| %></span><br>
+  <span>Title:<%= f.text_field :title %></span><br>
+  <span><% f.label :title %></span><br>
+  <span>Description:<%= f.text_field :description %></span><br>
+  <span><% f.label :description %></span><br>
+  <span><%f.label :Person %></span><br>
+  <span>Name:<%= f.select "person_id",
+  options_from_collection_for_select( @all_people , "id", "name"), :include_blank => true %></span><br><br>
+  <span> <%= f.submit %></span><br><br>
```

Comment:
Try not void using `<br />` tags as it can make styling harder to decipher. We usually don't use inline CSS for the same reason; having all your styles in one place within a .css file makes it much easier for others to work with your code, as well!